Comparing source compatibility of /home/someone/git/com.github/io7m/jregions/com.io7m.jregions.checkstyle/target/com.io7m.jregions.checkstyle-0.0.4.jar against /home/someone/.m2/repository/com/io7m/jregions/com.io7m.jregions.checkstyle/0.0.2/com.io7m.jregions.checkstyle-0.0.2.jar
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.checkstyle.Checkstyle  (not serializable)
